src/app/external-cluster-wizard/steps/summary/external-cluster-details/provider/aks-cluster-summary/template.html

```diff
@@ -0,0 +1,85 @@
+<!--
+Copyright 2022 The Kubermatic Kubernetes Platform contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<km-external-cluster-details *ngIf="spec && cloudSpec"
+                             [cluster]="cluster">
+  <ng-container leftColumn>
+    <km-property>
+      <div key>Location</div>
+      <div value>
+        <span>{{ cloudSpec.location }}</span>
+      </div>
+    </km-property>
+
+    <km-property>
+      <div key>Resource Group</div>
+      <div value>
+        <span>{{ cloudSpec.resourceGroup }}</span>
+      </div>
+    </km-property>
+  </ng-container>
+
+  <ng-container rightColumn>
+    <div class="section-header">Node Pool Setting</div>
```

Comment:
```suggestion
    <div class="section-header">Node Pool Settings</div>
```

Reply:
I'm following the wizard here, should I still make the change?
![Uploading image.png…]()